keps/0008-operator-testing.md

```diff
@@ -147,6 +147,8 @@ type TestSuite struct {
 	CRDDir            string
 	// Path to manifests to install before running tests.
 	ManifestsDir      string
+	// Paths to manifests to install before running tests.
+	ManifestsDirs     []string `json:"manifestsDirs"`
```

Comment:
do we need `ManifestsDirs` AND `ManifestsDir`?   can we remove the singular?

Reply:
I think that's three votes for dropping the singular (and therefore also losing backwards compatibility). I'll go ahead and drop it.